css/team.css

```diff
@@ -14,4 +14,21 @@
 .teammate_picture {
     border-radius: 45%;
     width: 250px;
+}
```

Comment:
Please use code formatter in VSC (right click on code and chose "Format Document").